modules/apps/segments/segments-web/src/main/resources/META-INF/resources/segments_experience_selector.jsp

```diff
@@ -28,6 +30,17 @@ SegmentsExperienceSelectorDisplayContext segmentsExperienceSelectorDisplayContex
 					<%= segmentsExperienceSelectorDisplayContext.getSelectedSegmentsExperienceName() %>
 				</span>
 
+				<%
+				boolean isSelectedExperienceActive = segmentsExperienceSelectorDisplayContext.isSelectedSegmentsExperienceActive(segmentsExperiencesJSONArray);
+				%>
+
+				<div class="autofit-col">
+					<span class="label label-<%= isSelectedExperienceActive?"success":"secondary" %> bg-transparent m-0">
+						<span class="label-item label-item-expand">
+							<%= segmentsExperienceSelectorDisplayContext.getSegmentsExperienceStatusLabel(isSelectedExperienceActive) %>
```

Comment:
As isSelectedExperienceActive is a value that is not going to change in the request, maybe we can return the value directly, without any parameter?